.github/dependabot.yml

```diff
@@ -11,6 +11,18 @@ updates:
         update-types: ['version-update:semver-patch']
       - dependency-name: 'cspell'
         update-types: ['version-update:semver-minor']
+      # ignore minor updates for eslint and any packages starting "eslint"
+      - dependency-name: "eslint*"
+        update-types: ['version-update:semver-minor']
+      # ignore minor updates for chai and any packages starting "chai"
+      - dependency-name: "chai*"
+        update-types: ['version-update:semver-minor']
```

Comment:
I don't think minor versions of chai should be ignored. There might be changes that break tests and waiting for the major release (which can bring other breaking changes) is not great.
```suggestion
```